communication/marketing-team/role-handbooks/editor.md

```diff
@@ -0,0 +1,71 @@
+# Editor Handbook
+
+## Overview
+
+The Editor takes on a beacon role for the group to bring multiple functions
+together
+
+Ensuring Kubernetes contributors have a wealth of robust, content and communications
+
+
+### Responsibilities
+
+- Facilitates meetings or can delegate to Associate Editor
+- Manage the editorial calendar and marketing project boards
+- Proofreading and editing articles with our [blogging] and [ethos] in mind including verifying proper quotes and approvals
+- Facilitate conversations and workflows between all types of stakeholders and partners:
+  - SIG Contributor Experience
+  - Community Group Chairs and Tech Leads  
+  - CNCF
+  - Steering Committee
+  - Contributors 
+- Working with the sig-docs liaison to manage communications and workflows
+between the groups
+- Develop high level marketing strategies and themes to address problem areas, challenges, and innovation/moving the project forward; expected to collaborate with Leads on their strategies for functional areas
+- Research and identify new and innovative ways of delivering content that
+appeals to our mission
+- Helping to recruit other members of the team, ensuring onboarding but delegating to proper lead; fall back when no lead present is the Editor
+- Ensure the team has a unified voice, focused, within the charter scope, and
+- Ability to modify the team charter and gain consensus with Contributor Experience communication [OWNERS] with major changes.
+- If time allows, blog about the team and it's progress, encourage team to blog about their work   
+
+### Minimum Skills and Requirements
+
+- Member of the [Kubernetes GitHub Org]
+- Shadowed under a lead in any role on the team for 6 months total (could be a combination of roles during that time period)
+- [Reviewer] in the marketing team folder  
+- Be ok with saying no but with a why/reason to back it up; be empathetic
+- Be comfortable with general GitHub workflows. We can teach you ours but a foundation for this role is necessary. You'll be working inside of multiple repositories with different experiences and helping others on the team with troubleshooting.
+- Have planned
+- Existing relationships with the contributor community either through other work or as an Associate Editor
+- Patience
+
+#### Expected Time Investment
+
+TODO  
+
+### Platforms, Tools, Access
+
+- Needs access to send and respond to email from community@kubernetes.io
+- GitHub - community and website repos, and project boards
+- kubernetes.io/blog
+- and everything listed in [team-resources.md]
+- what else?
+
+### Duration  
+
+6months to 12months??
+//discussion point
+
+## Associates Overview (Shadow)
+
+TODO
+
+### Expectations
+
+TODO  
+
+
+
+TODO
+Add relevant links
```

Comment:
```suggestion
# Editor Handbook

## Overview

The Editor takes on a beacon role for the group to bring multiple functions together ensuring Kubernetes contributors have a wealth of robust content and communications. 

### Responsibilities

- Develop high level marketing strategies and themes to address problem areas, challenges, and innovation/moving the project forward; expected to collaborate with Leads on their strategies for functional areas.
- Ensure the team has a unified voice, focus, and work within the charter scope.
- Research and identify new and innovative ways of delivering content that appeals to our mission
- Facilitates meetings or can delegate to Associate Editor.
- Manage the [editorial calendar](https://github.com/orgs/kubernetes/projects/41) and [marketing project](https://github.com/orgs/kubernetes/projects/39) boards.
- Proofreading and editing articles with our [guidlines] and [ethos] in mind.
- Act as or facilitate conversations and workflows between all types of stakeholders:
  - SIG Contributor Experience
  - Community Group Chairs and Tech Leads  
  - CNCF
  - Steering Committee
  - Contributors
- Working with liaison team members to manage communications and workflows between the groups.
- Helping to recruit other members of the team, ensuring onboarding but delegating to proper lead; fall back when no lead present is the Editor.
- Has the ability to modify the team charter and seeks consensus with Contributor Experience communication [OWNERS] when considering any major changes.
- If time allows, market the team; blog about the team and its progress and encourage the team to blog about their work.

### Minimum Skills and Requirements

Skills: 

- Be comfortable with general GitHub workflows. We can teach you ours but a foundation for this role is necessary. You'll be working inside of multiple repositories with different experiences and helping others on the team with troubleshooting.
- Strong written and verbal skills.
- Be ok with saying no, but with a reason to back it up; be empathetic.
- Have experience planning marketing activities for an organization.
- Have existing relationships with the Kubernetes contributor community either through other work or as an Associate Editor.
- Patience.

Requirements:

- Member in the [Kubernetes GitHub Org](https://github.com/kubernetes/community/blob/master/community-membership.md).
- [Reviewer] in the marketing team folder.
- Shadow under a lead in any role on the team for 6 months total (this could be a combination of roles during the time period).

#### Expected Time Investment

- Four hours a week as a minimal target.

### Platforms, Tools, Access

Here is a checklist of items we will work to get access when you accept the role.

- Access to send and respond to email from community@kubernetes.io
- Access to community and website repositories and project boards
- Other items listed in [team-resources.md]

### Duration  

6months to 12months??
//discussion point

## Associate Editors Overview (Shadow)

The Associate Editor should be able to achieve the above list of skills by the time they will be eligible to become the lead. 

### Expectations

Collaborate closely with the lead Editor toward the responsibilities outlined above. Be ready to backfill for them when they are unable to attend a meeting or represent the working group. 


See the [team resource page](https://github.com/kubernetes/community/blob/master/communication/marketing-team/team-resources.md) for relevant links. 
```